Loop/Managers/Alerts/UserNotificationAlertPresenter.swift

```diff
@@ -38,10 +38,8 @@ class UserNotificationAlertPresenter: AlertPresenter {
     }
     
     func retractAlert(identifier: Alert.Identifier) {
-        DispatchQueue.main.async {
-            self.userNotificationCenter.removePendingNotificationRequests(withIdentifiers: [identifier.value])
-            self.userNotificationCenter.removeDeliveredNotifications(withIdentifiers: [identifier.value])
-        }
+        self.userNotificationCenter.removePendingNotificationRequests(withIdentifiers: [identifier.value])
+        self.userNotificationCenter.removeDeliveredNotifications(withIdentifiers: [identifier.value])
```

Comment:
Not sure what this was on `DispatchQueue.main`. Seems unnecessary. Am I overlooking something?

Reply:
IIRC operations on UNUserNotificationCenter need to occur on `main`.  Have you tested it to make sure if it happens on another thread that there is no runtime error?  At any rate, IMHO I'd revert this change.